api/handlers.go

```diff
@@ -1208,6 +1239,7 @@ func handleContentUnits(db *sql.DB, r ContentUnitsRequest) (*ContentUnitsRespons
 	loadTables := []string{
 		"CollectionsContentUnits",
 		"CollectionsContentUnits.Collection",
+		"Tags",
```

Comment:
This is another DB query. It should be optional, i.e. a some request parameter. 
False be default as most requests hitting this endpoint do not need this information.